middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -937,9 +1185,20 @@ let close_one_function acc ~external_env ~by_closure_id decl
       ~dbg
       ~is_tupled
   in
+  let description = Env.{
+    fd_code_id = code_id;
+    fd_code = code;
+    fd_ret_cont = return_continuation;
+    fd_exn_cont = exn_continuation;
+    fd_params = params;
+    fd_body = body;
+    fd_closure = my_closure;
+    fd_depth = my_depth;
+  }
+  in
```

Comment:
Here, you should add a call to `Inlining_report.record_decision` to record the decision about the function declaration.